packages/riipen-ui/src/components/MenuItem.jsx

```diff
@@ -152,7 +152,8 @@ class MenuItem extends React.Component {
     );
 
     const handleClick = event => {
-      if (disabled) {
+      if (disabled && event) {
```

Comment:
It seems a bit strange that only sometimes an event is passed to the handleClick function?  Maybe something is going on in ListItem where it gets passed?

Reply:
it looks like event is always `undefined` -- I think it's because we are calling `handleClick` on the `onClick` of `ListItem`, and the `ListItem` component is wrapped in a `<React.Fragment>`?

I think this error hasn't popped up before because we don't seem to actually use the event in any of the `onClick` functions that are passed to `MenuItem`

I could only find `onClick` props that were functions that didn't use the `event` at all

Reply:
Nice sleuthing! 🕵️